_aggregations/pipeline/index.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: default
+title: Pipeline aggregations
+nav_order: 5
+has_children: true
+redirect_from:
+  - /opensearch/pipeline-agg/
+  - /query-dsl/aggregations/pipeline-agg/
+  - /query-dsl/aggregations/index/
```

Comment:
```suggestion
  - /query-dsl/aggregations/index/
  - /aggregations/pipeline/
```

Reply:
Line 10 is so you can omit "/index", and line 11 is so the old file redirects to the new one.